src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -948,10 +948,14 @@ public function patchEntriesAction(Entry $entry, Request $request, ContentProxy
             $entry->updateArchived((bool) $data['isArchived']);
         }
 
+        $entry->updateArchived($entry->isArchived());
```

Comment:
So here, the updateArchived() can be called twice?

Reply:
this addition is needed to make the test [`testSaveIsArchivedAfterPatch`](https://github.com/wallabag/wallabag/pull/6822/files#diff-b384b50ea4d69b72b2deb3d900a6da3a1f8d542c2df9abe0a442a08a669b0c5dR968-R993) pass.
if I understand it correctly, it's expected to have the archivedAt property filled/updated when a PATCH is made on an entry patching just another property when that entry has just the isArchived property to true but no archivedAt yet.
same for starred.
maybe it's the test expectation that is wrong?

Reply:
In fact the test ensure that when you patch an entries, the `is_archived` value doesn't change.
It was related to that issue: https://github.com/wallabag/wallabag/issues/2089

So I think you can remove the double call to `updateArchived` and revert changes on the test too.

Reply:
@j0k3r There must be a change in the test `testSaveIsStarredAfterPatch` at least, as on master it triggers a deprecation because `(new \DateTime($content['starred_at']))` is called with `$content['starred_at']` being null.
I could remove the commit "Fix entry PATCH API action" but I'd need to know what is the best way to fix that deprecation while still being aligned with what is expected :)

Hypothesis: maybe the issue is that the fixture entry that is fetched with `'isStarred' => true` condition has no starred at date?

Reply:
> Hypothesis: maybe the issue is that the fixture entry that is fetched with `'isStarred' => true` condition has no starred at date?

That's the problem: https://github.com/wallabag/wallabag/blob/c12239c4f8799259acd4beb34f0859d002877696/src/Wallabag/CoreBundle/DataFixtures/EntryFixtures.php#L77-L88

Reply:
@j0k3r fixed and rebased